### ISSUE TITLE
Not possible to pass custom cookie value

### ISSUE THREAD

**Issue by leathej1** [grafana/k6#1012]
I have a situation that has me 100% blocked. We have a cookie that we pass as a string (for reasons) and is formatted with surrounding quotes.

I am perfectly fine with manipulating the cookie object that you store and reformatting, but your code is trimming my quotes.

```
SSO Cookie: - - - {"myco_sso":["{'authToken':'WmHqwx2I.*AAJTwMQ..*'|'dataSessionTracker':'A3bSF/ysRd0JXdc='}"]} 

TYK Cookie: - - - "{'authToken':'WmHqwx2I.*AAJTwMQ..*'|'dataSessionTracker':'A3bSF/ysRd0JXdc='}" 

vuJar.set(response.url, "myco_sso", tykCookie, { domain: ".dev.myco.xyz", path: "/", secure: true, http_only: true });

INFO[0004] 2019/04/26 09:43:16 net/http: invalid byte '"' in Cookie.Value; dropping invalid bytes 

```

**Comment by leathej1** [comment]
I believe this is related to, but not exactly the same as https://github.com/loadimpact/k6/issues/711

**Comment by MStoykov** [comment]
this is definitely done by the [golang stdlib](https://github.com/golang/go/issues/10195) ... it is also what prints that error on the last line. I am of the opinion that the golang stdlib should probably return an error in this case instead of (not so silently) removing bytes from the values but that is beside the point. 
On the other hand `"` are **NOT**  allowed in cookie values and this should probably [not be depended upon by any application](https://github.com/golang/go/issues/10195#issuecomment-84758552). So whatever you are using to get and use this cookie value will/may/I hope *not* work in other situations ( probably some combination of OS + <browse name> will have problems using it).
That being said I did try to actually circumvent the golang stdlib in this [branch](https://github.com/MStoykov/k6/tree/dontTouchCookieValuesWhenMerging) of mine. I did however still didn't manage to get it to work with your script a week ago ... so ... :man_shrugging:
